### PR TITLE
YP Pref & Ch 1

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: youthful-piety/modernized/01-preface.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 ____
 Honourable age is not that which stands in length of time,
 nor that is measured by number of years, but wisdom is the gray hair unto men,
-and an unspotted life is old age.
+and an unspotted life is old age. -Wisdom
```
Comment (Henderjay) [inline youthful-piety/modernized/01-preface.adoc:5]: this quote (which I think is taken from the Wisdom of Solomon) didn't need you to put "--Wisdom" after it, becase the reference to Wisdom is already above in the epigraph tag: [quote.epigraph, , Wisdom].   So on the PDF it said Wisdom twice :) see pic.  
image

File: youthful-piety/original/01-preface.adoc [hunk 2]
```diff
@@ -2,7 +2,7 @@
 ____
 Honourable age is not that which stands in length of time,
 nor that is measured by number of years, but wisdom is the gray hair unto men,
-and an unspotted life is old age.
+and an unspotted life is old age. -Wisdom
```
Comment (Henderjay) [inline youthful-piety/original/01-preface.adoc:5]: same here.

File: youthful-piety/modernized/01-preface.adoc (hunk 1 above)
Comment (maijapapaya) [reply youthful-piety/modernized/01-preface.adoc:5]: ahhh!!! I didn't even notice that!! Ugh.

**Comment by Henderjay** [comment]
merged!

**Pull request merged**